sql-ballerina/tests/simple-params-query-test.bal

```diff
@@ -504,15 +504,15 @@ function queryDateStringInvalidParam() {
     }
 }
 
-@test:Config {
-    groups: ["query", "query-simple-params"]
-}
-function queryDateLongParam() {
-    // 1486080000000: 2017:02:03
-    DateValue typeVal = new (1486080000000);
-    ParameterizedQuery sqlQuery = `SELECT * from DateTimeTypes WHERE date_type = ${typeVal}`;
-    validateDateTimeTypesTableResult(queryMockClient(simpleParamsDb, sqlQuery));
-}
+//@test:Config {
+//    groups: ["query", "query-simple-params"]
+//}
+//function queryDateLongParam() {
+//    // 1486080000000: 2017:02:03
+//    DateValue typeVal = new (1486080000000);
+//    ParameterizedQuery sqlQuery = `SELECT * from DateTimeTypes WHERE date_type = ${typeVal}`;
```

Comment:
This should work when passing the int value directly to the query right? shall we change like that

Reply:
It seems we can not give int and save as date. I tried to save the date using int value but mistakenly used sql:DateValue rather than int type.